lib/packwerk/deprecated_references.rb

```diff
@@ -7,11 +7,16 @@ module Packwerk
   class DeprecatedReferences
     extend T::Sig
 
+    ENTRIES_TYPE = T.type_alias do
```

Comment:
In `parse_packwerk` (a gem Gusto put together to make it easier to read packwerk YML files), we have entries as their own first-class type. For now, I keep it as a hash with an unrefined type.